nodeJS/apis/project_blog_api.md

```diff
@@ -21,7 +21,7 @@ Why are we setting it up like this? Because we can! If you already have a portfo
 5. Create a second website for authoring and editing your posts. You can set this up however you like but the following features might be useful:
   - A list of all posts that shows whether or not they have been published.
   - A button to publish unpublished posts, or to unpublish published ones!
-  - A 'NEW POST' form. If you want to get fancy, you could use a rich text editor such as [TinyMCE](https://www.tiny.cloud/docs/quick-start/).
+  - A 'NEW POST' form. If you want to get fancy, you could use a rich text editor such as [TinyMCE](https://www.tiny.cloud/docs/tinymce/6/).
```

Comment:
If we could change the url to "https://www.tiny.cloud/docs/tinymce/6/cloud-quick-start/" I would be more than happy to accept it. Just so the URL still links to the quick-start page.